lib/phoenix/live_dashboard/components/nav_bar_component.ex

```diff
@@ -121,6 +127,9 @@ defmodule Phoenix.LiveDashboard.NavBarComponent do
         msg = ":render parameter must be in item: #{inspect(item)}"
         raise ArgumentError, msg
 
+      {component, component_assigns} when is_atom(component) and is_map(component_assigns) ->
+        item
+
```

Comment:
We don't allow this on purpose, otherwise we may initialize components that we don't use. You should always pass a function. :)

```suggestion
```